src/react/ISV/components/__tests__/ISVSteps.test.tsx

```diff
@@ -0,0 +1,209 @@
+import { render, screen } from '@testing-library/react';
+import * as React from 'react';
+import { ISVStepperContext, useISVStepper } from '../ISVSteps';
+import { ThemeProvider } from 'styled-components';
+import {
+  VEEAM_BACKUP_REPLICATION_XML_VALUE,
+  VEEAM_OFFICE_365,
+} from '../../constants';
+import { ISVPlatformConfig } from '../../types';
+
+jest.mock('../ISVSteps', () => {
+  const originalModule = jest.requireActual('../ISVSteps');
+
+  return {
+    ...originalModule,
+    ISVSteps: jest.fn().mockImplementation(() => null),
+  };
+});
+
+// Mock dependencies
+jest.mock('react-router', () => ({
+  useSearchParams: jest.fn(),
+}));
+
+// Mock Scality core-ui components
+jest.mock('@scality/core-ui', () => ({
+  spacing: { r16: '16px' },
+  Stepper: ({ steps }) => (
+    <div data-testid="stepper">
+      {steps.map((step, index) => (
+        <div key={index} data-testid={`step-${index}`}>
+          {step.label}
+        </div>
+      ))}
+    </div>
+  ),
+  Text: ({ children }) => <div>{children}</div>,
+}));
+
+jest.mock('@scality/core-ui/dist/next', () => ({
+  Box: ({ children }) => <div data-testid="box">{children}</div>,
+}));
+
```

Comment:
Core-ui shouldn't be mocked out. Mocking with data-testid is a bad practice that I discourage because it coupled the unit test with the current implementation.